tests/backends/tests.py

```diff
@@ -970,9 +970,7 @@ def runner():
             t.start()
             t.join()
         # Each created connection got different inner connection.
-        self.assertEqual(
-            len(set(conn.connection for conn in connections_dict.values())),
-            3)
+        self.assertEqual(len({conn.connection for conn in connections_dict.values()}), 3)
```

Comment:
Sure, I just found the original version more readable. There is a soft limit on 79 chars meaning if it looks fine you can break.